webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBarActions.js

```diff
@@ -19,7 +19,7 @@ export const closeSwitcher = () => ({
 
 export const loadSwitcherResourcesByResource = (resource, options = {}) => (dispatch) => {
   const { resourceUrl, nameField, switcherItemUrl } = resource;
-  const { page = 1 } = options;
+  const { page = 1, search } = options;
```

Comment:
You need to update the code for calculating `pages` too. I should use `subtotal` (number of items matching your query) instead of `total` (number of all items).